activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/QueryTasksIT.java

```diff
@@ -81,7 +81,9 @@
 
     private static final String TASKS_URL = "/v1/tasks";
     private static final String ADMIN_TASKS_URL = "/admin/v1/tasks";
-
+    private static final String HRUSER = "hruser";
```

Comment:
I think a test should rely only on fake usernames

Reply:
we use `testuser` in several ITs in activiti cloud for that purpose, in order not to pollute test code with specific use cases

Reply:
In my understanding, only hruser and testuser are configurated to access the APIs. When using other usernames tests fails with 401 error. I need 2 users for the test I want to cover.